src/ReservoirPriceOracle.sol

```diff
@@ -385,6 +395,7 @@ contract ReservoirPriceOracle is IPriceOracle, IReservoirPriceOracle, Owned(msg.
             _getRouteDecimalDifferencePrice(lToken0, lToken1);
 
         if (lRoute.length == 0) {
+            // REVIEW: Is it possible for `aQuote` to be an IERC4626?
```

Comment:
nope, I've asked this qn to Euler several weeks ago. We can control the "unit of account" of the debt. So we can always set it to the underlying asset. 

So I've assessed that in our operational cases we'll not encounter the case where `aQuote` is an IERC4626.

Reply:
Kk

Reply:
I don't think we have to worry about this case. The routes defined are in the underlying assets.

We technically could add a check when setting routes to make sure that the start and end tokens are not ERC4626, but I think that's unnecessary

Reply:
KK, can remove review coment and merge